src/Components/OATHeader/internal/ModalSaveCurrentProjectAndClear.tsx

```diff
@@ -66,9 +69,9 @@ export const ModalSaveCurrentProjectAndClear = ({
 
             <div className={headerStyles.modalRowCenterItem}>
                 <Text styles={promptTextStyles}>
-                    {`${t('OATHeader.doYouWantToSaveChangesYouMadeTo')} ${
-                        project.projectName
-                    }?`}
+                    {`${t(
+                        'OATHeader.doYouWantToSaveChangesYouMadeTo'
+                    )} ${projectName}?`}
```

Comment:
We should be using interpolation in scenarios like this one: https://www.i18next.com/translation-function/interpolation

Some languages might have `projectName` in different places within the string so this provides additional flexibility.